src/findboundaries.jl

```diff
@@ -0,0 +1,22 @@
+"""
+```
+bordered_img = findboundries(img)
+bordered_img = findboundries(img, background)
+```
+Returns a copy of the original image where boundries between labeled regions are true.
+Parameters:
+ -  img             = Binary/Grayscale input image
+ -  background      = Value to be given to pixels/elements that are cleared (Default value is 0)
+"""
+
+function findboundaries(img::AbstractArray, background::Integer=0)
```

Comment:
Perhaps this function should be called `find_boundaries` as per the convention of splitting words with underscores?